README.md

```diff
@@ -326,88 +326,7 @@ onLCP(sendToAnalytics);
 
 ### Send the results to Google Analytics
 
-Google Analytics does not support reporting metric distributions in any of its built-in reports; however, if you set a unique dimension value (in this case, the metric `id`, as shown in the examples below) on every metric instance that you send to Google Analytics, you can create a report yourself using the [Google Analytics Reporting API](https://developers.google.com/analytics/devguides/reporting) and any data visualization library you choose.
```

Comment:
Overall I think that intro content is still important to keep, especially the bit about GA not supporting distributions (and thus no p75). I just want to make sure we're using the correct terminology. I'm 99% sure the Data API can do this, what I'm not clear on is whether or not the API can query the event param directly or if you need to register a custom dimension in the UI first.

Note: I'm not suggesting we figure this all out in this PR, but I also want to make sure we're not saying anything incorrect. What about something like:

> Google Analytics does not support reporting metric distributions in any of its built-in reports; however, if you set a unique event parameter value (in this case, the `metric_id`, as shown in the example below) on every metric instance that you send to Google Analytics, you can create a report yourself by first getting the data via the [Google Analytics Data API](https://developers.google.com/analytics/devguides/reporting/data/v1) or via [BigQuery export](https://support.google.com/analytics/answer/9358801) and then visualizing it any charting library you choose.

Reply:
LGTM!